src/feature/nodelist/routerinfo.c

```diff
@@ -17,14 +17,31 @@
 #include "feature/nodelist/node_st.h"
 #include "feature/nodelist/routerinfo_st.h"
 
-/** Copy the primary (IPv4) OR port (IP address and TCP port) for
- * <b>router</b> into *<b>ap_out</b>. */
+/** Copy the OR port (IP address and TCP port) for <b>router</b> and
+ * <b>family</b> into *<b>ap_out</b>.
+ *
+ * If the requested ORPort does not exist, sets *<b>ap_out</b> to the null
```

Comment:
This API is a bit strange to me.  Why not also return 0 or -1 depending on whether the port exists?

Reply:
Sure, see c76f310fe724c3d93416b4e23a219feda8e6f217.

I've possibly been writing too much Rust, where errors are part of the returned type :-)